chart/epinio-installer/templates/manifest.yaml

```diff
@@ -0,0 +1,264 @@
+            value: {{ default "admin" .Values.user | quote }}
+          - name: "password"
+            value: {{ default $password .Values.password | quote }}
+          - name:  "certManagerNamespace"
+            value: cert-manager
+          - name: "server.sessionKey"
+            value: "{{ $sessionKey }}"
+          - name: "server.accessControlAllowOrigin"
+            value: {{ default "*" .Values.accessControlAllowOrigin | quote }}
+          - name: "server.timeoutMultiplier"
+            value: "1"
+          - name: "server.traceLevel"
+            value: "1"
+
+          - name: "s3.endpoint"
+            value: {{ default "minio.minio-epinio.svc.cluster.local" .Values.s3Endpoint | quote }}
+          - name: "s3.bucket"
+            value: {{ default "epinio" .Values.s3Bucket | quote }}
+          - name: "s3.accessKeyID"
+            value: {{ default $accessKey .Values.s3AccessKeyId | quote }}
+          - name: "s3.secretAccessKey"
+            value: {{ default $secretKey .Values.s3SecretAccessKey | quote }}
+          - name: "s3.region"
+            value: {{ default "" .Values.s3Location | quote }}
+          - name: "s3.useSSL"
+            value: {{ default "false" .Values.s3UseSSL | quote }}
+
+          - name: "registry.url"
+            value: {{ default (print "epinio-registry." .Values.domain)  .Values.externalRegistryURL | quote }}
+          - name: "registry.localhostURL"
+            value: "127.0.0.1:30500"
+          - name: "registry.username"
+            value: "{{ $registryUsername }}"
+          - name: "registry.password"
+            value: "{{ $registryPassword }}"
+          - name: "registry.namespace"
+            value: {{ default "apps" .Values.externalRegistryNamespace | quote }}
+          - name: "registry.certificateSecret"
+            value: "epinio-registry-tls"
+          - name: "registry.certificateSecretNamespace"
+            value: "epinio-registry"
+          - name: "server.forceKubeInternalRegistryTLS"
+            value: {{ default "false" .Values.forceKubeInternalRegistryTLS | quote }}
+        preDeploy:
+          - type: "pod"
+            selector: "app=tekton-pipelines-webhook"
+            namespace: tekton-pipelines
+          - type: "crd"
+            selector: "clustertasks.tekton.dev"
+          - type: "crd"
+            selector: "conditions.tekton.dev"
+
+      {{ if not .Values.externalRegistryURL -}}
+      - id: namespace-registry
+        {{- if not .Values.skipLinkerd }}
+        needs: linkerd
+        {{- end }}
+        type: namespace
+        namespace: epinio-registry
+        values:
+          - name: linkerd.io/inject
+            value: enabled
+            type: annotation
+
+      - id: registry
+        # needs: namespace-registry
+        needs: cert-manager
+        namespace: epinio-registry
+        type: helm
+        source:
+          name: epinio-registry
+          path: assets/installer/container-registry-0.1.0.tgz
```

Comment:
This now has a release: https://github.com/epinio/epinio-helm-chart/releases/tag/container-registry-0.1.0

Maybe you can add it here: https://github.com/epinio/epinio-helm-chart/pull/19/files#diff-2e442565fbce952d11e55f84855064fdd5e9d981d922df9af960f7b6ef733e22R54

Reply:
@jimmykarily is it ok if I remove the file from the installer and switch the chart to the URL?

Reply:
done